### PR TITLE
Refactor XXX releases fetching support

### PR DESCRIPTION
**Pull request by DariusIII**
Addresses issue #.
No XXX info pulled from some of the sources, should fix issue #2472 
Changes made by this pull request.
- Refactor XXX releases support
- Fix matching to all of the sources

### CODE REVIEW

File: app/extensions/helper/helpers.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
```
Comment (niel) [inline app/extensions/helper/helpers.php:1]: This is in the wrong place. lithium extensions are for extending a base lithium class.
Put this in \nzedb\utilities\misc.php for now

File: composer.json [hunk 2]
```diff
@@ -5,6 +5,9 @@
     }
```
Comment (niel) [inline composer.json:5]: This won't be needed after moving the function to  \nzedb\utilities\misc.php

File: app/extensions/helper/helpers.php (hunk 1 above)
Comment (DariusIII) [reply app/extensions/helper/helpers.php:1]: I have used laravel logic, we do have getUrl in misc and i did not want to mix the two.

Comment (niel) [reply app/extensions/helper/helpers.php:1]: so rename yours?
 ;-)

Comment (DariusIII) [reply app/extensions/helper/helpers.php:1]: Mine is getRawHtml :)

**Comment by niel** [comment]
Closed for Darius to create a new PR.

**Pull request closed without merging**